R/label-dollar.R

```diff
@@ -126,3 +162,15 @@ dollar <- function(x, accuracy = NULL, scale = 1, prefix = "$",
 
   amount
 }
+
+#' @export
+#' @rdname label_dollar
+short_scale <- function() {
```

Comment:
I think these should have a common prefix rather than a common suffix. `scales::scale_short()` seems a bit confusing though. I don't have any better ideas right now, but maybe you do?

Reply:
How about if they're called `rescale_short_scale()` and `rescale_long_scale()`? It's a bit verbose, but it aligns with the argument name (`rescale_large`). It also leaves room for things like `rescale_indian()` etc.

Reply:
Yeah, I like it.